Autopilot/PathManager/Src/AutoSteer.cpp

```diff
@@ -12,14 +12,15 @@
  * Variables
  **********************************************************************************************************************/
 
-static PIDController bankPid{1, 0, 0, 0, -MAX_BANK_ANGLE, MAX_BANK_ANGLE}; // PID gains need to be tuned
-static PIDController rudderPid{1, 0, 0, 0, -100, 100}; // PID gains need to be tuned
+static PIDController bankPid{4.2,0,0,0, -MAX_BANK_ANGLE, MAX_BANK_ANGLE}; //5,0,8,0,
 
-static PIDController pitchPid{1, 0, 0, 0, -MAX_PITCH_ANGLE, MAX_PITCH_ANGLE}; // PID gains need to be tuned
+static PIDController rudderPid{0.2,0.2,7,1, -100, 100}; //0.2,0.2,7,1
 
-static PIDController altitudePid{1, 0, 0, 0, -100, 100}; //PID gains need to be tuned
+static PIDController pitchPid{1,0,0,0, -MAX_PITCH_ANGLE, MAX_PITCH_ANGLE}; //1,0,0,0,
 
-static const float RUDDER_SCALING_FACTOR = 0.8f; // should be experimentally determined
+static PIDController altitudePid{110,0,0,0, 0, 100}; //110,0,0,0,
```

Comment:
this seems massive. A single meter off would drive the throttle to 0 or 100%. That's an incredible strain on the motor.

Reply:
yup will change. Name should be changed too, since we restructured it to be throttle controlling airspeed.